test/unit/specs/methods.spec.js

```diff
@@ -1,14 +1,45 @@
 const AppleSignin = require('ti.applesignin');
+const signInButton = AppleSignin.createLoginButton({
+	type: AppleSignin.BUTTON_TYPE_SIGNIN,
+	style: AppleSignin.BUTTON_STYLE_BLACK,
+	borderRadius: 10,
+});
 
 describe('ti.applesignin', () => {
 	describe('methods', () => {
 		describe('#authorize()', () => {
 			it('is a function', () => {
 				expect(AppleSignin.authorize).toEqual(jasmine.any(Function));
 			});
-			// TODO: verify return value is one of the RESULT_* constants!
```

Comment:
@vikas-goyal-axway, is this TODO is not valid now **TODO: verify return value is one of the RESULT_* constants!**, i do not see it is addressed.

Reply:
@fasihul-axway , We can not check it as this function need user input and we don't have any thing to track user input in UT

Reply:
Okay @vikas-goyal-axway, then please remove this TODO in a specific commit with above message.